### PR TITLE
Include DOB in referral template.

### LINKED ISSUE #7824: Include DOB in referral template
### Describe the problem
Many providers expect a patient's date of birth when accepting a referral.  It is not included in the referral form, however.

### Describe the solution you'd like
Include the DOB on the referral form.

### PR DESCRIPTION
**Pull request by ChrisOstler**
Fixes #7824

#### Short description of what this resolves:
Add the date of birth to the referral form, as it is expected by many providers.

#### Does your code include anything generated by an AI Engine? No

### CODE REVIEW

File: interface/patient_file/transaction/print_referral.php [hunk 1]
```diff
@@ -28,6 +28,7 @@
   'label_webpage_title'         => xlt('Referral Form'),
   'label_form1_title'           => xlt('Referral Form'),
   'label_name'                  => xlt('Name'),
+  'label_dob'                   => xlt('DOB'),
```
Comment (sjpadgett) [inline interface/patient_file/transaction/print_referral.php:31]: Recommend adding locale formatting for DOB to match your locate date format setting in Config.
To do use the`oeFormatShortDate` function in the foreach/conditional around L-195.
E.g.
```
foreach ($patdata as $key => $value) {
    if ($key == "DOB") {
        $s = str_replace("{pt_$key}", text(oeFormatShortDate($value)), $s);
    }
    if ($key == "sex") {
        $s = str_replace("{pt_$key}", generate_display_field(array('data_type' => '1','list_id' => 'sex'), $value), $s);
    } else {
        $s = str_replace("{pt_$key}", text($value), $s);
    }
}
````

**Review by sjpadgett — CHANGES_REQUESTED** [review]
Looks good and we appreciate this submission. I only have one change request otherwise ready to merge.
I'll look into your notification PR soon

File: interface/patient_file/transaction/print_referral.php (hunk 1 above)
Comment (ChrisOstler) [reply interface/patient_file/transaction/print_referral.php:31]: Thanks!

**Review by sjpadgett — APPROVED** [review]
Merging Thanks for the contribution.

**Pull request merged**